src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0;i < 3;i++) {
+            System.out.println(lottery.getRandomBall().toString());
+        }
```

Comment:
All magic numbers in your code should be constants.
Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.
Make code prettifier, add spaces after ;
The toString method is automatically called when something is printed using println. You don't need to call this method in `System.out.println`